opsgenie_access/templates/opsgenie_access/access_request_form.html

```diff
@@ -1,96 +1,231 @@
-<div class="page-header">
-  <div class="container-fluid">
-    <h2 class="h5 no-margin-bottom">{{ each_access.formDesc }}</h2>
-  </div>
-</div>
+<script>
 
-<input type="hidden" id="opsgenieLabel" name="accessLabel" value="[]">
-<div class="row">
-  <div class="col-md-12">
-    <div class="row">
-      <div class="col-sm-3">
-        <label for="Access" style="font-weight: bolder; min-height:0px; margin-top:10px; font-size:medium;">Access Level:</label>
-        <select id="opsgenieAccessLevel" name="opsgenieAccessLevel" class="form-control" required>
-          <option hidden disabled selected value> -- select -- </option>
-          <option value="user" >Member</option>
-          <option value="team_admin" >Admin</option>
-        </select>
-      </div>
-    </div>
+  const selectedList = {}
```

Comment:
change this variable something specific to opsgenie
```suggestion
  const opsgenieSelectedList = {}
```

Reply:
done.